pkg/platform/model/inventory.go

```diff
@@ -51,17 +51,19 @@ func IngredientByNameAndVersion(language, name, version string, prefix Namespace
 	}
 
 	for _, ingredient := range results {
-		if ingredient.Ingredient.Name == nil || *ingredient.Ingredient.Name != name {
-			continue
-		}
+		for _, feature := range ingredient.LatestVersion.ProvidedFeatures {
+			if feature.Feature == nil || strings.ToLower(*feature.Feature) != strings.ToLower(name) {
```

Comment:
Maybe, let's not yet add the case-insensitive part here...  I have just opened a PR that we can work on after this one lands: https://github.com/ActiveState/cli/pull/1121  I had to park that on Friday because the Ctrl-C story was prioritized.

It involves one more step: preferring the exact casing if both options are available.